cmd/interlock/run.go

```diff
@@ -35,7 +35,7 @@ var cmdRun = cli.Command{
 		cli.StringFlag{
 			Name:  "config, c",
 			Usage: "path to config file",
-			Value: "",
+			Value: "/etc/interlock/config.toml",
```

Comment:
This also means that the Docker command needs to be explicit which I think is also better (explicit vs. implicit) for configuration.